HMH-README.md

```diff
@@ -0,0 +1,23 @@
+### Building
+
+This must be built from a linux machine, will not work on m1 mac.
+
+#### 0.12.x
+```
+git clone https://github.com/pveasey/consul-k8s
```

Comment:
Should the pveasey's (in a few places) change to hmh?

Reply:
done